pkg/network/config/configtest.go

```diff
@@ -0,0 +1,23 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package config
+
+import (
+	"strings"
+	"testing"
+
+	aconfig "github.com/DataDog/datadog-agent/pkg/config"
+)
+
+// ResetConfig resets the configuration.
+func ResetConfig(t *testing.T) {
```

Comment:
Why isn't this in `config_test.go`?

Reply:
He is using it outside of this package.

Reply:
I think this function belongs elsewhere, because it is affecting `config.SystemProbe`, which is not a network-specific object. Maybe `pkg/config/test_helpers.go` and name it `ResetSystemProbeConfig`?

Reply:
done